src/map/lua/lua_baseentity.cpp

```diff
@@ -18861,6 +18861,150 @@ void CLuaBaseEntity::claimContestReward()
     }
 }
 
+/************************************************************************
+ *  Function: createLamp()
```

Comment:
Can we have the name be verbose?
As in `createEinherjarLamp`

There are many lamps. We have nyzul using lamps, old dynamis used some similar thing... Just to make it immediately clear what this is used for

Reply:
Speaking with Claywar, it seems this functions arent needed, since we have ways to set exdata in lua
`player:addItem({ id = itemID, exdata = { [10] = 10 } }) -- add 1 item of itemID, with the exdata at index 10 (0-indexed!) set to 10`